packages/config/config/devices/0x027a/templates/zooz_template.json

```diff
@@ -732,6 +732,64 @@
 			}
 		]
 	},
+	"external_switch_type_0-12": {
+		"$import": "#external_switch_type_0-3",
+		"maxValue": 12,
+		"options": [
+			{
+				"label": "Automatic detection",
+				"value": 0
+			},
+			{
+				"label": "Toggle switch (state changes whenever the switch is toggled)",
+				"value": 1
+			},
+			{
+				"label": "Momentary push button",
+				"value": 2
+			},
+			{
+				"label": "Toggle switch (light on when the switch is toggled up, light off when the switch is toggled down)",
+				"value": 3
+			},
+			{
+				"label": "Leak alarm (water sensor)",
+				"value": 4
+			},
+			{
+				"label": "Heat alarm",
+				"value": 5
+			},
+			{
+				"label": "Motion alert",
+				"value": 6
+			},
+			{
+				"label": "Open/Close alert (door sensor)",
+				"value": 7
+			},
+			{
+				"label": "CO alarm",
+				"value": 8
+			},
+			{
+				"label": "CO2 alarm",
+				"value": 9
+			},
+			{
+				"label": "On/Off report (dry contact switch/sensor)",
```

Comment:
If we're being strict, an "On/Off report" is not a switch type. I know the manual calls it this way, but we should be somewhat consistent. Maybe change the label to "Input type / Report type" or something like that?

Reply:
fixed